specs/WinRT/MakeMSIX-spec.md

```diff
@@ -0,0 +1,1002 @@
+        if (exitCode == EXIT_FAILURE)
+        {
+            winrt::throw_hresult(E_FAIL);
+        }
+    }
+
+    void Example_SignPackage(std::wstring pfxFile, std::wstring packageFile)
+    {
+        std::wstring signToolArguments;
+        signToolArguments.append(L"sign /fd SHA256 /f ");
+        signToolArguments.append(pfxFile);
+        signToolArguments.append(L" " + packageFile);
+        LaunchSignToolWithArguments(signToolArguments);
+    }
+```
+
+# 3. API Details
+
+```c#
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+namespace Microsoft.Kozani.MakeMSIX
+{
+    [contractversion(1)]
+    apicontract MakeMSIXContract{};
+
+    /// Options for creating a package
+    [contract(MakeMSIXContract, 1)]
+    runtimeclass CreatePackageOptions
+    {
+        /// Created by callers to pass options into CreatePackage()
+        CreatePackageOptions();
+
+        /// Optional replacement package name.
+        /// If field is empty, value from AppxManifest.xml is unchanged.
+        String Name;
+
+        /// Optional replacement package publisher.
+        /// If field is empty, value from AppxManifest.xml is unchanged.
+        String Publisher;
+
+        /// Optional replacement package version
+        /// If version is 0, value from AppxManifest.xml is unchanged.
+        Windows.ApplicationModel.PackageVersion Version;
+        
+        /// Overwrite output file if it already exists.
+        /// Defaults to true.
+        Boolean OverwriteOutputFileIfExists;
+
+        /// Validates elements of the package during creation.
+        /// Defaults to false.
+        Boolean ValidateFiles;
+    };
+
+    /// Options for creating a Kozani package
+    [contract(MakeMSIXContract, 1)]
+    runtimeclass CreateKozaniPackageOptions
+    {
+        /// Created by callers to pass options into CreateKozaniPackage()
+        CreateKozaniPackageOptions();
+
+        /// Remove extensions from the manifest.
+        /// Defaults to false.
+        Boolean RemoveExtensions;
+
+        /// Optional replacement package name.
+        /// If field is empty, value from AppxManifest.xml is unchanged.
+        String Name;
+
+        /// Optional replacement package publisher.
+        /// If field is empty, value from AppxManifest.xml is unchanged.
+        String Publisher;
+
+        /// Optional replacement package version
+        /// If version is 0, value from AppxManifest.xml is unchanged.
+        Windows.ApplicationModel.PackageVersion Version;
+
+        /// Languages supported by the output package
+        /// If field is empty, all languages from the original package are maintained.
+        Windows.Foundation.Collections.IVector<String> Languages{ get; };
+
+        /// ScaleFactors supported by the output package
+        /// If field is empty, all scale factors from the original package are maintained.
+        Windows.Foundation.Collections.IVector<UInt32> ScaleFactors{ get; };
```

Comment:
This refers to e.g. `<Package><Resources><Resource Scale="150">`?

Are there other `<Resource>`s deserving the same support?

[Docs](https://learn.microsoft.com/uwp/schemas/appxpackage/uapmanifestschema/element-resource) note `<Resource>` also supports Language and DXFeatureLevel